app/assets/stylesheets/config/_bootstrap_variables.scss

```diff
@@ -23,3 +23,4 @@ $border-radius-lg: 2px;
 $border-radius-sm: 2px;
 
 // Override other variables below!
+body { background: #ffffff !important;}
```

Comment:
The recommended way of doing this would be to add a new color in config/colors and then changing body-bg in this file.

Reply:
Is there a reason why that is the recommended way? this way works just fine ...

Reply:
The two links at the top of this file (in the comments) explain how scss  variables are declared and overriden. 

In this case you're basically overriding body-background colour 2 times, first on line 9 and  then again on line 26.

Reply:
Looking at the bootstrap variables again, I realised that the default bootstrap body-bg color is already white. So in this case, we don't need to override anything. Simply commenting out line 9 will work.

Reply:
I will try Anjali suggestion and merge